README.md

```diff
@@ -1,2 +1,31 @@
 # hw01
-Repository for first CFSS homework assignment
+Repository for first CFSS homework assignment      
+Author: Michelle (Ronghan) Wang     
+Date: Jan 4, 2017   
+
+# Biography
+### Michelle (Ronghan) Wang | MAPSS
+![photo](https://media.licdn.com/mpr/mpr/shrinknp_400_400/AAEAAQAAAAAAAAdYAAAAJDAxNzRmMzNjLTRlYmItNDM4ZC05NDcyLTE1MjMwNDZkNmIxZA.jpg)
+
+Hi I'm **Michelle Wang**! I'm a current *MAPSS* student with a *Psychology* concentration here at the _University of Chicago_. I with work Dr. [Ayelet Fishbach](http://faculty.chicagobooth.edu/ayelet.fishbach/research/) in her lab of behavioral science research in Booth. I received my B.A. in *psychology* and *French studies* from Colby College in 2016.
+
+## Interest
+* Photography
+* Travel
+* Sports
+    + Biking
+    + Skiing
+* Cooking
+* Watching Films
+
+## Research Interest
+* Human Behavioral Science
+* Social Psychology
+* Behavioral Marketing
+
+### Website
+Please check out my personal website on [LinkedIn!](https://www.linkedin.com/in/michelle-ronghan-wang-45ab2677)
+
+#### Pictures
+Here's a picture of me in the Grand Canyon National Park
+!(http://ww1.sinaimg.cn/mw690/81dda96dgw1fbfrbs7jnsj21kw16oqk1.jpg)
```

Comment:
The last picture isn't working for me - I think it's because there are no brackets before the URL in parentheses.